examples/cartopy_convert.py

```diff
@@ -0,0 +1,105 @@
+# ---------------------
+#
+# Cartopy also handles Shapely objects well, but it uses a different system for
+# CRS. To plot this data with CartoPy, we'll first need to project it into a
+# new CRS. We'll use a CRS defined within CartoPy and use the GeoPandas
+# ``to_crs`` method to make the transformation.
+
+# Define the CartoPy CRS object.
+crs = ccrs.AzimuthalEquidistant()
+
+# This can be converted into a `proj4` string/dict compatible with GeoPandas
+crs_proj4 = crs.proj4_init
+df_ae = df.to_crs(crs_proj4)
+
+# Here's what the plot looks like in GeoPandas
+df_ae.plot()
+
+###############################################################################
+# Now that our data is in a CRS based off of CartoPy, we can easily
+# plot it.
+
+fig, ax = plt.subplots(subplot_kw={'projection': crs})
+ax.add_geometries(df_ae['geometry'], crs=crs)
+
+###############################################################################
+# Note that we could have easily done this with an EPSG code like so:
+crs_epsg = ccrs.epsg('3857')
+df_epsg = df.to_crs(epsg='3857')
+
+# Generate a figure with two axes, one for CartoPy, one for GeoPandas
+fig, axs = plt.subplots(1, 2, subplot_kw={'projection': crs_epsg},
+                        figsize=(10, 5))
+# Make the CartoPy plot
+axs[0].add_geometries(df_epsg['geometry'], crs=crs_epsg,
+                      facecolor='white', edgecolor='black')
+# Make the GeoPandas plot
+df_epsg.plot(ax=axs[1], color='white')
+
+###############################################################################
+# CartoPy to GeoPandas
+# --------------------
+#
+# Next we'll perform a CRS projection in CartoPy, and then convert it
+# back into a GeoPandas object.
+
+crs_new = ccrs.AlbersEqualArea()
+new_geometries = [crs_new.project_geometry(ii, src_crs=crs)
+                  for ii in df_ae['geometry'].values]
+
+fig, ax = plt.subplots(subplot_kw={'projection': crs_new})
+ax.add_geometries(new_geometries, crs=crs_new)
+
+###############################################################################
+# Now that we've created new Shapely objects with the CartoPy CRS,
+# we can use this to create a GeoDataFrame.
+
+df_aea = gpd.GeoDataFrame(df['gdp_pp'], geometry=new_geometries,
+                          crs=crs_new.proj4_init)
+df_aea.plot()
+
+###############################################################################
+# We can even combine these into the same figure. Here we'll plot half
+# of the countries with CartoPy, and the other half with GeoPandas. For
+# the GeoPandas countries, we'll color each based based on a feature
+# we've calculated above.
+
+# Generate a CartoPy figure and add half the countries to it
+fig, ax = plt.subplots(subplot_kw={'projection': crs_new})
+ax.add_geometries(new_geometries[:len(new_geometries) // 2], crs=crs_new)
+# Make the GeoPandas plot
+df_aea.iloc[len(new_geometries) // 2:].plot(column='gdp_pp', ax=ax,
+                                            vmin=0, vmax=.02)

```

Comment:
I personally find this plotting of half/half the geometries with cartopy and geopandas makes the example more difficult to understand (the `[:len(new_geometries) // 2]` is additional and not necessary boilerplate to understand). 
Maybe we can plot two different things? 
Eg plot all of the countries with geopandas `.plot`, and then add eg the capitals with cartopy, or the oceans.